api.swagger.yaml

```
@@ -437,6 +437,20 @@ paths:
         '403':
           description: 'User is not authorized to view this.'
       description: Returns a count of users who accessed the site on 4 or more days within a given calendar month.
+  /users/metrics/profiles:
+    get:
+      tags:
+        - users
+      responses:
+        '200':
+          description: 'Count of users profile metrics.'
```

This is still unclear to me. Is this what is meant?
```suggestion
          description: 'User count and profile metrics.'
```

---

Or perhaps this?
```suggestion
          description: 'Counts of user profiles matching various conditions.'
```